layouts/404.html

```diff
@@ -0,0 +1,18 @@
+{{ define "main"}}
+    <main id="main" style="border-left-style: none;border-right-style: none;">
+      <div class="container">
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <div class="row">
+          <div class="col-12" style="text-align: center;">
+            <h1 id="title">404</h1>
+            <p>Oops! This page doesn't exist.</p>
+            <a href="{{ "/" | relURL }}" class="btn btn-outline-secondary home-btn">Back to Home</a>
```

Comment:
Were we going to have an image on this one?
What do design say?

Reply:
The images already available didn't really seem suitable, we can go over them tomorrow.